symfony/framework-bundle/4.2/config/services.yaml

```diff
@@ -15,7 +15,11 @@ services:
     # this creates a service per class whose id is the fully-qualified class name
     App\:
         resource: '../src/*'
-        exclude: '../src/{DependencyInjection,Entity,Tests,Kernel.php}'
+        exclude:
+            - '../src/DependencyInjection/*'
```

Comment:
Do we need the asterisk here? Would `../src/DependencyInjection/` work?

If I'm right, in `glob`, a single `*` means contents in that dir only and double `**` means "recursive" and goes into all subdirectories.

So `../src/DependencyInjection/**` could be more precise, but `../src/DependencyInjection/` looks easier to understand to me. But maybe I'm confusing things.

Reply:
I don't know :man_shrugging:  /cc @nicolas-grekas

Reply:
Indeed, the `*` is not mandatory. I just checked.

Should I remove it?

Reply:
In the `.gitignore` file, other Symfony recipes add entries like this:

```
/public/bundles/
/var/
/vendor/
```

Instead of this:

```
/public/bundles/*
/var/*
/vendor/*
```

So, if it's technically correct, removing the asterisk would be consistent with other Flex features. Thanks.

Reply:
@javiereguiluz Done, But now we have an inconsistency with the `resource`:

```
    App\:
        resource: '../src/*'
        exclude:
            - '../src/DependencyInjection/'
            - '../src/Entity/'
            - '../src/Kernel.php'
            - '../src/Tests/'

    # controllers are imported separately to make sure services can be injected
    # as action arguments even if you don't extend any base controller class
    App\Controller\:
        resource: '../src/Controller'
        tags: ['controller.service_arguments']
```

So I fixed that too :)